### PR TITLE
BGDIINF_SB-2447: Fix the kgs mako to run with py2 and py3

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

File: Makefile.frankfurt [hunk 1]
```diff
@@ -58,6 +58,9 @@ DOCKER_REGISTRY = 974517877189.dkr.ecr.eu-central-1.amazonaws.com
 AWS_REGION_ECR := eu-central-1
 AUTHOR=$(USER)
 
+# data.geo.admin.ch hosts
+DATAGEOADMINHOST ?= data.geo.admin.ch
```
Comment (ltshb) [inline Makefile.frankfurt:62]: I think as default we should use DEV staging for local work ?

File: chsdi/templates/htmlpopup/kgs.mako [hunk 2]
```diff
@@ -17,21 +17,23 @@
...
-        try:
-            from urllib2 import urlopen
-        except ImportError:
-            # Python3 fallback
-            from urllib.request import urlopen
         dataGeoAdminHost = request.registry.settings['datageoadminhost']
         csv_url = "https://" + dataGeoAdminHost + "/" + c['layerBodId']  + "/image/meta.txt"
         csv_file = None
         try:
+            from urllib2 import urlopen
```
Comment (ltshb) [inline chsdi/templates/htmlpopup/kgs.mako:24]: Add a TODO cleanup python2

File: Makefile.frankfurt (hunk 1 above)
Comment (ltshb) [reply Makefile.frankfurt:62]: Just for my understanding, what has this env variable to do with the mako and this fix ?

Comment (rebert) [reply Makefile.frankfurt:62]: right now there is only one data.geo.admin.ch - once, this will be replaced with stac ...

Comment (rebert) [reply Makefile.frankfurt:62]: The images showed in the tooltip are located on data.geo.admin.ch (and right now only there):
https://mf-chsdi3.prod.bgdi.ch/rest/services/ech/MapServer/ch.babs.kulturgueter/9980/extendedHtmlPopup?lang=en

Comment (rebert) [reply Makefile.frankfurt:62]: :-) and the csv being filtered by featureID as well on data.geo.admin.ch

File: chsdi/templates/htmlpopup/kgs.mako [hunk 3]
```diff
@@ -17,21 +17,23 @@
         objarts = c['attributes']['objektart'].split(',')
         gemeinde_ehemalig = c['attributes']['gemeinde_ehemalig'] if c['attributes']['gemeinde_ehemalig'].startswith("(") else "("+c['attributes']['gemeinde_ehemalig']+")"
         import csv
-        try:
-            from urllib2 import urlopen
-        except ImportError:
-            # Python3 fallback
-            from urllib.request import urlopen
         dataGeoAdminHost = request.registry.settings['datageoadminhost']
```
Comment (rebert) [inline chsdi/templates/htmlpopup/kgs.mako:20]: @ltshb here the link to the env variable

**Review by ltclm — APPROVED** [review]

**Review by ltshb — APPROVED** [review]
LGTM just adds the `TODO python2 cleanup` in order to find them easier later on.

**Pull request merged**